ext/src/ruby_api/engine.rs

```diff
@@ -18,6 +18,25 @@ lazy_static::lazy_static! {
 
 /// @yard
 /// Represents a Wasmtime execution engine.
+///
+/// @example Disabling parallel compilation
+///    # Many Ruby servers use a pre-forking mechanism to allow parallel request
+///    # processing. Unfortunately, this can causes processes to deadlock if you
+///    # use parallel compilation to compile Wasm prior to calling
+///    # `Process::fork`. To avoid this issue, any compilations that need to be
```

Comment:
```suggestion
///    # `Process.fork`. To avoid this issue, any compilations that need to be
```

While totally valid, the `::` notation for class method feels less common than the `.` notation. No strong opinion though!